src/wp-admin/options-general.php

```diff
@@ -155,16 +155,27 @@
 	}
 	?>
 
-
-	<div id="site-icon-preview" class="site-icon-preview wp-clearfix settings-page-preview <?php echo esc_attr( $classes_for_wrapper ); ?>">
-		<div class="favicon-preview">
-			<img src="<?php echo esc_url( admin_url( 'images/' . ( is_rtl() ? 'browser-rtl.png' : 'browser.png' ) ) ); ?>" class="browser-preview" width="182" alt="">
-			<div class="favicon">
-				<img id="browser-icon-preview" src="<?php site_icon_url(); ?>" alt="<?php echo esc_attr( $browser_icon_alt_value ); ?>">
+	<style>
+	:root{
+		--site-icon-url: url( '<?php site_icon_url(); ?>' );
```

Comment:
`site_icon_url()` called multiple time can we store it value in variable and reuse it?